include/quicly/cc.h

```diff
@@ -0,0 +1,38 @@
+/* Interface definition for quicly's congestion controller.
+ */
+
+#ifndef quicly_cc_h
+#define quicly_cc_h
+
+#include "quicly/constants.h"
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+
+struct ccstate {
```

Comment:
Should we prefix the structs with `quicly_` prefix now that we have our own congestion controller?

Also, I think we might want to change the struct to
```c
typedef struct st_quicly_cc_state_t {
    ...
} quicly_cc_state_t;
```
or something. The conventions are:
* structs are named `st_<name>_t`
* external structs are typedef, with the `st_` prefix being omitted.

Or we could simply name it `quicly_cc_t` omitting "state".

Reply:
That's great to know! I've made the changes. To clarify, "external structs" means structs that are declared and used in different files (including .h)?

Reply:
> To clarify, "external structs" means structs that are declared and used in different files (including .h)?

Correct.